forwarding-history/randomize-data-fast.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Check if the CSV file location is provided
+if [ -z "$1" ]; then
+  echo "Error: Please provide the CSV file location as an argument."
+  exit 1
+fi
+
+# Read the CSV file
+csv_file="$1"
+
+# Validate if the file exists
+if [ ! -f "$csv_file" ]; then
+  echo "Error: File '$csv_file' not found."
+  exit 1
+fi
+
+# Output file
+output_file="forwarding_data.csv"
+
+# Temporary file to store random values
+random_values_file=$(mktemp)
+
+# Write header to the output file
+echo "timestamp_ns,chan_id_in,chan_id_out,amt_in_msat,amt_out_msat,fee_msat" > "$output_file"
+
+# Number of random values to pre-generate
+random_pool_size=1000000
+
+# Function to generate random values and append to the temporary file
+generate_random_pool() {
+  shuf -i 100000000000000000-999999999999999999 -n $random_pool_size >> "$random_values_file"
+}
+
+# Generate initial pool of random values
+generate_random_pool
+
+# Use awk to process the CSV file
+awk -F, -v output_file="$output_file" -v random_values_file="$random_values_file" -v random_pool_size="$random_pool_size" '
+BEGIN {
+  counter = 0
+  rv_idx = 0
+  random_value_count = 0
+  while ((getline < random_values_file) > 0) {
+    random_values[random_value_count++] = $0
+  }
+  close(random_values_file)
+}
+function get_next_random() {
```

Comment:
One of the goals for randomization is to have a consistent mapping for `channel_id` (in/out) -> `random_value` - we want to hide the `channel_id` because it contains information about the channel but we want to keep the traffic pattern the same so that we can look at the amount of htlcs that are being forwarded over a specific channel. 

If I'm reading this correctly, we're not doing that mapping like we do in the other randomization script so we'll lose the traffic patterns here?